CHANGELOG.md

```diff
@@ -44,6 +44,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 
 ### Improvements
 
+- [#182](https://github.com/babylonlabs-io/babylon-sdk/pull/182) e2e: fix e2e tests
```

Comment:
maybe we can add no-changelog to skip this meaningless changelog

Reply:
i'd say we can keep this changelog as it does some meaningful changes in e2e tests. Wdyt?